Source/cmNinjaNormalTargetGenerator.cxx

```diff
@@ -463,13 +465,25 @@ void cmNinjaNormalTargetGenerator::WriteLinkRule(bool useResponseFile,
       this->GetLocalGenerator()->CreateRulePlaceholderExpander());
 
     // Rule for linking library/executable.
-    std::vector<std::string> linkCmds = this->ComputeLinkCmd(config);
+    std::vector<std::string> linkCmds =
+      this->ComputeLinkCmd(config, commands, ranlibIndexes);
+
+    // To get cmake commands, uncomment it
```

Comment:
What is the problem here? Is there a way to unify it without any commenting / uncommenting?

Reply:
If we wanna see cmake commands in JSON, we will uncomment it. For now, we don't need this. I can delete this line

Reply:
return pair